src/index.html

```diff
@@ -6,8 +6,58 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="card" data-qa="card">
+
+      <img
+        src="images/imac.jpeg"
+
+        alt="APPLE A1419 iMac 27' Retina 5K Monoblock (MNED2UA/A)"
+
+        >
+
+      <h2 class="card__info">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <section class="card__code">Product code: 195434</section>
+
+      <section class="card__reviws">
+        <div class="stars stars--4">
+          <mark class="stars__star"></mark>
+          <mark class="stars__star"></mark>
+          <mark class="stars__star"></mark>
+          <mark class="stars__star"></mark>
+          <mark class="stars__star"></mark>
+        </div>
+
+        <div class="card__reviws-count">Reviews: 5</div>
+
+      </section>
+
+      <section class="card__price">
+        <div class="card__price-lable">Price:</div>
+        <div class="card__price-how">$2,199</div>
```

Comment:
use semantic tags for text

Reply:
<img width="624" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/55d435a1-0536-4c75-91fc-2a82510ad067">

Reply:
still not fixed, you can just use `p` or `span` tags